bin/runyarntests.sh

```diff
@@ -22,6 +22,10 @@ export REEF_TEST_YARN=true
 DEPENDENCY_JAR=`echo $REEF_HOME/lang/java/reef-tests/target/reef-tests-*-test-jar-with-dependencies.jar`
 CLASSPATH=`yarn classpath`
 
-CMD="java -cp $YARN_CONF_DIR:$DEPENDENCY_JAR:$CLASSPATH org.junit.runner.JUnitCore org.apache.reef.tests.AllTestsSuite $*" 
+if [ $1 = "org.apache.reef.tests.yarnnodelabel.NodeLabelTest" ]; then
```

Comment:
No, I am not 😄
@scan3 please don't reference specific tests in the generic script. I will open a separate JIRA ticket to update the script in a minute. The fix should look like this:
```bash
CMD="java -cp $YARN_CONF_DIR:$DEPENDENCY_JAR:$CLASSPATH org.junit.runner.JUnitCore ${*:-org.apache.reef.tests.AllTestsSuite}"
```
(or one can use more sophisticated command line options etc.)

I'll send the PR for the script soon.